--- firefox-49.0.1/toolkit/components/telemetry/healthreport-prefs.js~	2016-06-01 06:11:41.000000000 +0200
+++ firefox-49.0.1/toolkit/components/telemetry/healthreport-prefs.js	2016-10-02 09:39:52.367936470 +0200
@@ -4,7 +4,7 @@
 
 pref("datareporting.healthreport.infoURL", "https://www.mozilla.org/legal/privacy/firefox.html#health-report");
 
-// Health Report is enabled by default on all channels.
-pref("datareporting.healthreport.uploadEnabled", true);
+// Health Report is disabled by default on all channels.
+pref("datareporting.healthreport.uploadEnabled", false);
 
 pref("datareporting.healthreport.about.reportUrl", "https://fhr.cdn.mozilla.net/%LOCALE%/v4/");
