--- firefox-60.8.0/tools/profiler/core/platform.h.orig	2019-08-06 21:57:41.190763126 +0200
+++ firefox-60.8.0/tools/profiler/core/platform.h	2019-08-06 22:00:13.309983976 +0200
@@ -46,8 +46,10 @@
 // wrapper for it.
 #if defined(__GLIBC__)
 #include <unistd.h>
+#if !__GLIBC_PREREQ(2, 30)
 #include <sys/syscall.h>
 static inline pid_t gettid() { return (pid_t)syscall(SYS_gettid); }
+#endif
 #elif defined(GP_OS_darwin)
 #include <unistd.h>
 #include <sys/syscall.h>
